src/components/common/Talk/styles.js

```diff
@@ -0,0 +1,118 @@
+export default theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    marginBottom: 20
```

Comment:
Don't set margins on components. This should be defined in the parent component.